docs/rst/tutorials/core/qos/xml_profiles/xml_profiles.rst

```diff
@@ -0,0 +1,409 @@
+.. include:: ../../../../exports/alias.include
+
+.. _tutorials_xml_profiles_intro:
+
+Configuring Fast-DDS QoS via XML profiles
+=========================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+Vulcanexus offers the possibility of fully configuring Fast DDS' QoS policies through XML profile definition.
+This can be done in several ways depending on whether the application used is a native Fast DDS application or a ROS 2 application using Fast DDS's RMW implementation.
+This guide will focus on the latter.
+For more information regarding XML profiles in Fast DDS please refer to the
+`XML configuration page <https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/xml_configuration.html>`_
+from Fast DDS's documentation.
```

Comment:
For the entire document, do not break lines unless `.`
```suggestion
For more information regarding XML profiles in Fast DDS please refer to the `XML configuration page <https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/xml_configuration.html>`_ from Fast DDS's documentation.
```

Reply:
Odd line break there. Fixed.